test/manifold_test.cpp

```diff
@@ -160,6 +160,31 @@ TEST(Manifold, Decompose) {
   RelatedGL(manifolds, input);
 }
 
+TEST(Manifold, DecomposeProps) {
+  auto posNorm = [](float* newProp, glm::vec3 pos, const float* oldProp) {
```

Comment:
Actually, maybe just replace this with our test function `WithPositionColors`. That's also convenient for debug, since you can export the meshes and look at the coloration to figure out what's going on.

Reply:
Done